src/webview/direct-connect-form.ts

```diff
@@ -55,62 +55,36 @@ class DirectConnectFormViewModel extends ViewModel {
     }
   }
 
-  async testConnection(event: Event) {
+  /** Submit all form data to the extension host */
+  async handleSubmit(event: SubmitEvent) {
     event.preventDefault();
     this.success(false);
-    this.errorMessage("");
-    // const form = event.target as HTMLFormElement;
-    // const formData = new FormData(form);
-    // const data = Object.fromEntries(formData.entries());
-    // TODO not implemented yet
-    // const result = await post("TestConnection", data);
-    // if (result.success) {
-    //   console.log("success", result);
-    // } else {
-    //   this.errorMessage(result.message ?? "Unknown error occurred");
-    //   console.log("error", result);
-    // }
-  }
-
-  /** Submit all form data to the extension */
-  async handleSubmit(event: Event) {
-    event.preventDefault();
-    this.success(false);
-    this.errorMessage("");
+    this.message("");
+    this.loading(true);
     const form = event.target as HTMLFormElement;
+    const submitter = event.submitter as HTMLInputElement;
     const formData = new FormData(form);
+
     const data = Object.fromEntries(formData.entries());
     if (!data["bootstrap_servers"] && !data["uri"]) {
-      return this.errorMessage("Please provide either Kafka cluster or Schema Registry details");
-    }
-    let clusterConfig: KafkaClusterConfig | undefined = undefined;
-    let schemaConfig: SchemaRegistryConfig | undefined = undefined;
-    if (data["bootstrap_servers"]) {
-      clusterConfig = transformFormDataToKafkaConfig(data);
-    }
-    if (data["uri"]) {
-      schemaConfig = transformFormDataToSchemaRegistryConfig(data);
+      this.message("Please provide either Kafka cluster or Schema Registry details");
```

Comment:
How do I get out of this state? 😅 
<img width="640" alt="image" src="https://github.com/user-attachments/assets/291fc1d5-f7fc-4ccd-ad45-09f182cf3f8f" />

Reply:
ahh good catch 😅  I will fix that